docs/topics/mpp/mpp-connect-to-apis.md

```diff
@@ -72,7 +72,30 @@ internal actual fun writeLogMessage(message: String, logLevel: LogLevel) {
 The main rules regarding expected and actual declarations are:
 * An expected declaration is marked with the `expect` keyword; the actual declaration is marked with the `actual` keyword.
 * `expect` and `actual` declarations have the same name and are located in the same package (have the same fully qualified name).
-* `expect` declarations never contain any implementation code.
+* `expect` declarations never contain any implementation code and are abstract by default.
```

Comment:
"abstract" has a particular meaning in Kotlin. Is it used in this meaning here?

Reply:
It is, meaning 'the one that doesn't have an implementation'. Like in [Abstract classes](https://kotlinlang.org/docs/classes.html#abstract-classes) or [Interfaces](https://kotlinlang.org/docs/interfaces.html#properties-in-interfaces)

Reply:
I suppose it would be confusing, because "abstract" usually means abstract modality. 
`expect` declarations can be explicitly declared `abstract` or not (e.g. `open`, `final`). Being expect doesn't imply being abstract.

Reply:
Not sure that if it's not explicitly marked as `abstract` means it's not abstract

@dsavvinov, @koshachy What do you think? Should I change the wording here?